### PR TITLE
Add Anomaly task to OTX 1.6.x

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have ran e2e tests and there is no issues.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

**Comment by chuneuny-emily** [comment]
Please do not merge this PR into releases/1.6.0 branch. This PR branch was just merged into aurelien/CVS-147615-Anomaly-reduction-documentation-update branch in Geti for early work on Geti side. Since this feature is targeting Geti 2.5, it should be merged in OTX2.2.x.

File: src/otx/cli/manager/config_manager.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "ACTION_CLASSIFICATION": "Custom_Action_Classification_X3D",
     "ACTION_DETECTION": "Custom_Action_Detection_X3D_FAST_RCNN",
     "VISUAL_PROMPTING": "Visual_Prompting_SAM_ViT_B",
+    "ANOMALY": "otx_anomaly_padim",
```
Comment (samet-akcay) [inline src/otx/cli/manager/config_manager.py:46]: this is defined here but never used anywhere. Do you think if this is needed? Or the other model template ids updated?

**Review by samet-akcay — APPROVED** [review]

**Comment by samet-akcay** [comment]
Same as #3910, the detection tests are failing here. Any thoughts @harimkang ?

**Comment by harimkang** [comment]
> Same as #3910, the detection tests are failing here. Any thoughts @harimkang ?

I'm not sure about 1.6. @chuneuny-emily Can you give us some advice?

**Pull request closed without merging**